rtm/plotting.py

```diff
@@ -177,6 +187,21 @@ def plot_time_slice(S, processed_st, time_slice=None, label_stations=True,
     if dem is not None:
         ax.set_aspect('equal')
 
+    ax_pos = ax.get_position()
+    cloc = [ax_pos.x1+.02, ax_pos.y0, .02, ax_pos.height]
+    cbaxes = fig.add_axes(cloc)
+    cbar = fig.colorbar(sm, cax=cbaxes, label='Stack amplitude')
+    cbar.solids.set_alpha(1)
+
+    if plot_peak:
+        s_peak = S.max(axis=(1, 2)).data
+
+        ax1.plot(S.time, s_peak, 'k-')
+        ax1.set_xlim(S.time[0].data, S.time[-1].data)
+        ax1.set_aspect('auto')
+        ax1.set_xlabel('UTC Time')
+        ax1.set_ylabel('Peak Stack Amplitude')
```

Comment:
I think this makes it a little more clear?
```suggestion
        ax1.set_ylabel('Stack amplitude at spatial stack max')
```